### ISSUE TITLE
Improve UX and accessibility of buttons/icons

### ISSUE THREAD

**Issue by dwolfe** [hypothesis/product-backlog#856]
Reported by @xolotl in Slack:
![screen shot 2018-10-30 at 2 29 02 pm](https://user-images.githubusercontent.com/1348738/47751891-320ce380-dc50-11e8-88a9-1b118c3fa50c.png)

The current sizes of the social icons at the bottom right of the client are below the recommendations for mobile applications, and accessibility more generally. The icons themselves are 16px tall and 13.5-16px wide (borders added below to the `<i>`, `<button>`, and `<annotation-action-button>` elements for visibility:

![screen shot 2018-10-30 at 2 12 17 pm](https://user-images.githubusercontent.com/1348738/47752236-24a42900-dc51-11e8-9d35-95272f7f1a17.png)

WCAG AAA compliance requires a minimum size of 44px by 44px, and Google recommends 48px by 48px with 8px spacing. More here:

- https://www.w3.org/TR/WCAG21/#target-size
- https://developers.google.com/web/fundamentals/accessibility/accessible-styles

A minimal remedy could be to increase the size of these icons to 44px tall, and extend the hit area to 44px wide; but ideally we should develop a robust, reusable, responsive button component that we can leverage here and elsewhere. Such a button component would ideally also include a text label, and adapt to only show the text label when a lack of space requires it (See https://www.nngroup.com/articles/icon-usability/).

**Comment by dwhly** [comment]
FWIW I am a strong advocate for being able to unflag-- but appreciate the larger Q of control size.

**Comment by dwolfe** [comment]
@ajpeddakotla If this is still relevant, could you open a new issue for it? Thanks!

**Comment by ajpeddakotla** [comment]
Closed in favor of #1085
